docs/README.md

```diff
@@ -1,58 +1,82 @@
 
-**Send pull requests to the master branch, not gh-pages.  gh-pages will be overwritten by deploy without warning.**
+To contribute to the documentation, open a GitHub pull request.
 
+## Key Guidelines for Contributors
 
-## Prerequisites
+- You need to fork this repository.
+- Work on your fork's task branches, not the `master` branch.
+- Send pull requests to the `master` branch in this repository.
+- If you want to submit a knowledge base article, see [Submitting a KB Article](#submitting-a-kb-article).
+- Follow the [ScyllaDB Style Guide](https://docs.google.com/document/d/1lyHp1MKdyj0Hh3NprNFvEczA4dFSZIFoukGUvFJb9yE/edit?usp=sharing).
+
+The user documentation is written in reStructuredText (RST) - a plaintext markup language similar to Markdown. If you're not familiar with RST, see [ScyllaDB RST Examples](https://sphinx-theme.scylladb.com/stable/examples/index.html).
+
+The developer documentation is written in Markdown. See [Basic Markdown Syntax](https://www.markdownguide.org/basic-syntax/) for reference.
+
+
+## Submitting a KB Article
+
+If you are submitting a knowledgebase article (KB), use the following guidelines:
+* In the `/kb_common` directory, there is a template for KBs. It is called `kb-article-template.rst`.
+* Make a copy of the KB template in the `/kb` directory and rename it with a unique name.
+* Open the new file and fill in the required information. 
+* Remove what is not needed. 
+* Run `make preview` to build the docs and preview them locally.
+* Send a PR with "KB" in its title. 
+
+
+## Building User Documentation
+
+### Prerequisites
 
 * Python 3. Check your version with `$ python --version`
-* Vale CLI (optional to lint docs). [Install Vale](https://docs.errata.ai/vale/install) for your operative system.
+* Vale CLI (optional to lint docs). [Install Vale](https://docs.errata.ai/vale/install) for your OS.
 
-### Prerequisites: Mac OS X
+#### Mac OS X
 
 You must have a working [Homebrew](http://brew.sh/) in order to install the needed tools.
 
-You also need the standard utility `make`.  (I don't know if this comes with Mac OS X.)
+You also need the standard utility `make`.
 
-Check if you have these two items with
+Check if you have these two items with the following commands:
 
 ```sh
 brew help
 make -h
 ```
 
-### Prerequisites: Fedora 29/Debian-based Linux Distributions
+#### Fedora 29/Debian-based Linux Distributions
 
-This should work out of the box with Fedora 29.
+Building the user docs should work out of the box.
 
-### Prerequisites: Windows
+#### Windows
 
-Use "Bash on Ubuntu on Windows", everything should be same as on a debian-based Linux.
-Note: livereload seems not working on Windows.
+Use "Bash on Ubuntu on Windows" for the same tools and capabilities as on a debian-based Linux.
 
-## Prerequisites: other systems
+Note: livereload seems not to be working on Windows.
```

Comment:
@tzach Is this relevant?

Reply:
No idea. @dgarcia360 please advise

Reply:
This is not an issue anymore, we can delete the warning.

Reply:
Removed